src/CommunityToolkit.Maui/Extensions/Navigation/NavigationExtensions.netstandard.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Threading.Tasks;
+using CommunityToolkit.Maui.Views;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+
+namespace CommunityToolkit.Maui.Extensions;
+
+public static partial class NavigationExtensions
```

Comment:
actually, we can remove this file at all. it is not used

Reply:
No we can't